### ISSUE TITLE
Sloboda class to issue stream update when worker is assigned to building.

### ISSUE THREAD

**Issue by gladimdim** [gladimdim/sloboda#5]
Producible class has 'changes' stream. When the worker is added to the building the update is put into the stream. Sloboda class must listen to all streams from all buildings and issue its own changes event in order for StreamBuilders to rebuild the widget tree with new info.

If the building is destroyed the appropriate stream must be closed.

An idea is to merge all Producible changes streams into one stream and listen to it. If the building is removed, the stream will be auto removed from that merged stream. If the building is added, Sloboda must merge into stream into existing.

**Comment by gladimdim** [comment]
Once this is done, we have to remove all setState statements which did the Widget update call manually.

**Comment by gladimdim** [comment]
The CityBuilder widget was introduced into the components. Use it to auto update the widgets underneath it. No implementations of new subscription is needed as it was already present in the Sloboda logic.
